File: Pokémon 3D Server/Pokémon 3D Server/Class/Setting/Data List/SwearInfractionList.vb
Public Class SwearInfractionList
    Public Property Name As String
    Public Property GameJoltID As Integer
    Public Property Points As Integer
    Public Property Muted As Integer
    Public Property StartTime As Date

    Public Sub New(ByVal Name As String, ByVal GameJoltID As Integer, ByVal Points As Integer, ByVal Muted As Integer, ByVal StartTime As Date)
        Me.Name = Name
        Me.GameJoltID = GameJoltID
        Me.Points = Points
        Me.Muted = Muted
        Me.StartTime = StartTime
    End Sub
End Class

File: Pokemon_3D_Server_Client/Pokemon_3D_Server_Core/Settings/Data List/SwearInfractionList.cs
using System;

namespace Global
{
    /// <summary>
    /// Class containing SwearInfractionList data
    /// </summary>
    public class SwearInfractionList : CommonList
    {
        /// <summary>
        /// New SwearInfractionList
        /// </summary>
        /// <param name="Name">Player Name</param>
        /// <param name="GameJoltID">Player GameJolt ID</param>
        /// <param name="Points">Player Infraction Points</param>
        /// <param name="Muted">Player Mute Points</param>
        /// <param name="StartTime">Player Infraction StartTime</param>
        public SwearInfractionList(string Name,int GameJoltID,int Points,int Muted,DateTime StartTime)
        {
            this.Name = Name;
            this.GameJoltID = GameJoltID;
            this.Points = Points;
            this.Muted = Muted;
            this.StartTime = StartTime;
        }
    }
}
